src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -84,7 +84,7 @@ limitations under the License.
       <arg name="Certificate" type="OCTET_STRING"/>
     </command>
 
-    <command source="client" code="0x04" name="OpCSRRequest" optional="false">
+    <command source="client" code="0x04" name="OpCSRRequest" response="OpCSRResponse" optional="false">
```

Comment:
This is `CSRRequest`/`CSRResponse` (without `Op` prefix) by spec

Reply:
Good catch.  https://github.com/project-chip/connectedhomeip/issues/14759 to deal with that.